packages/kmod-6.12-nvidia-r570/kmod-6.12-nvidia-r570.spec

```
@@ -569,6 +654,28 @@ popd
 # peermem
 %{_cross_datadir}/nvidia/open-gpu/drivers/nvidia-peermem.ko
 
+# GRID driver files
+%if "%{_cross_arch}" == "x86_64"
+%files grid
+%license COPYING
+%dir %{_cross_datadir}/nvidia/grid/drivers
+%dir %{_cross_factorydir}/nvidia/grid
+
+%{_cross_datadir}/nvidia/grid/drivers/nvidia.ko
+
+# uvm
+%{_cross_datadir}/nvidia/grid/drivers/nvidia-uvm.ko
+
+# modeset
+%{_cross_datadir}/nvidia/grid/drivers/nvidia-modeset.ko
+
+# drm
+%{_cross_datadir}/nvidia/grid/drivers/nvidia-drm.ko
+
+# peermem
+%{_cross_datadir}/nvidia/grid/drivers/nvidia-peermem.ko
```

The comments don't add much value, so you could consolidate these fields:
```suggestion
%{_cross_datadir}/nvidia/grid/drivers/nvidia.ko
%{_cross_datadir}/nvidia/grid/drivers/nvidia-uvm.ko
%{_cross_datadir}/nvidia/grid/drivers/nvidia-modeset.ko
%{_cross_datadir}/nvidia/grid/drivers/nvidia-drm.ko
%{_cross_datadir}/nvidia/grid/drivers/nvidia-peermem.ko
```